### PR TITLE
Add audiodecoders

### PR DESCRIPTION
**Pull request by notspiff**
This adds
- a sap audio decoder
- a gme audio decoder
- a gsf audio decoder
- a ncsf audio decoder
- an organya audio decoder
- a qsf audio decoder
- a ssf audio decoder
- a wsr audio decoder
- an upse audio decoder
- a fluidsynth audio decoder (+ lib as static)
- a libopenmpt audio decoder (+ lib as static)
- a dumb audio decoder
- an usf audio decoder

### CODE REVIEW

File: packages/audio/fluidsynth/package.mk [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+#  along with OpenELEC.  If not, see <http://www.gnu.org/licenses/>.
+################################################################################
+
+PKG_NAME="fluidsynth"
+PKG_VERSION="1.1.6"
+PKG_REV="1"
+PKG_ARCH="any"
+PKG_LICENSE="GPL"
+PKG_SITE="http://fluidsynth.org/"
+PKG_URL="$DISTRO_SRC/$PKG_NAME-$PKG_VERSION.tar.bz2"
```
Comment (stefansaraev) [inline packages/audio/fluidsynth/package.mk]: PKG_URL="$SOURCEFORGE_SRC/project/fluidsynth/fluidsynth-$PKG_VERSION/$PKG_NAME-$PKG_VERSION.tar.bz2"
so we dont have to mirror it.

**Comment by stefansaraev** [comment]
usf fails on rpi (1) http://sprunge.us/gWcd

what about moving those to unofficial repo btw ?

**Comment by notspiff** [comment]
this is your turf, i do as requested.

**Comment by notspiff** [comment]
lazyusf requires simd, neon or sse.

**Comment by stefansaraev** [comment]
ic. then I wont build it for rpi1 :)

**Comment by stefansaraev** [comment]
all built. thanks much.

ping @sraue

**Comment by notspiff** [comment]
fixup squashed.

**Comment by sraue** [comment]
@stefansaraev we can add to our normal repo, not sure how "unofficial" this is, we also dont expect to much updates over the time? @notspiff when this addons will be used normally? and will they be merged to official kodi?

**Comment by sraue** [comment]
@stefansaraev can we exit 0 or so if TARGET_CPU="arm1176jzf-s" for the usf addon? or better if SIMD_SUPPORT="no" see config/arch.arm

**Comment by stefansaraev** [comment]
> we can add to our normal repo, not sure how "unofficial" this is

sure. that's easier for me, we keep it there.

> can we exit 0 or so if TARGET_CPU="arm1176jzf-s"

we can,I'll take care later. for now it's one liner in my repo buildscript

**Comment by notspiff** [comment]
srau whenever you want to play some music in these formats. Also I plan to do a tunes addon for retro like there is for tvshows.

**Pull request merged**